xgi/algorithms/centrality.py

```diff
@@ -249,3 +250,61 @@ def node_edge_centrality(
     return {node_dict[n]: new_x[n] for n in node_dict}, {
         edge_dict[e]: new_y[e] for e in edge_dict
     }
+
+
+def vector_centrality(H):
+    """
+    Compute the vector centrality of nodes in hypergraphs
+
```

Comment:
Tiny suggestion, for uniformity with other docstrings elsewhere
```suggestion
    """The vector centrality of nodes in the line graph of the hypergraph.
```